src/tools/erasure-code/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_executable(ceph-erasure-code-tool
+  ${PROJECT_SOURCE_DIR}/src/osd/ECUtil.cc
+  ceph-erasure-code-tool.cc)
+target_link_libraries(ceph-erasure-code-tool global ceph-common)
+install(TARGETS ceph-erasure-code-tool DESTINATION bin)
```

Comment:
as this tool is installed, we will have to package it otherwise rpmbuild would complain as we have something installed but not packaged.

@trociny how about packaging it in "ceph-test"? i understand this tool falls into the category of rescue-kit. but we don't have such a package yet. the closest one is "ceph-test".

Reply:
I'd love if this tool came together with ceph-objectstore-tool, that is in the ceph-osd package. Though I understand it does not so perfectly fit the ceph-osd package as the ceph-objectstore-tool. So I don't have a strong opinion here -- if people think it is better to have this in ceph-test package, I will add it here.

Reply:
agreed. it should be colocated with ceph-osd along with COT.